src/ngx_http_lua_socket_tcp.c

```diff
@@ -4908,65 +4901,30 @@ ngx_http_lua_socket_push_input_data(ngx_http_request_t *r,
     nbufs = 0;
     ll = NULL;
 
+    luaL_buffinit(L, &luabuf);
     for (cl = u->bufs_in; cl; cl = cl->next) {
+        ptrdiff_t chunk_size;
```

Comment:
Could you clarify why it's better to use `size_t` to compute a difference between two pointers?

Could you point this style rule?

Thanks.